src/dynamichmc_inference.jl

```diff
@@ -6,10 +6,11 @@ struct DynamicHMCPosterior
     kwargs
 end
 
+logpdf(dist,data::Flux.Tracker.TrackedReal) = Distributions.logpdf(dist,data.data)
```

Comment:
no, don't drop the derivatives on the logpdf. This will lead to incorrect derivatives. If you are directly using Flux at all, there's a good change it's wrong.

Reply:
But it won't work without this dispatch?

Reply:
The second test in dynamicHMC.jl?